consensus/XDPoS/utils/errors.go

```diff
@@ -86,5 +86,5 @@ type ErrIncomingMessageRoundNotEqualCurrentRound struct {
 }
 
 func (e *ErrIncomingMessageRoundNotEqualCurrentRound) Error() string {
-	return fmt.Sprintf("%s message round number: %v does not match currentRound: %v", e.Type, e.IncomingRound, e.CurrentRound)
+	return fmt.Sprintf("%s message round number: %v is more than 1 round away from currentRound: %v", e.Type, e.IncomingRound, e.CurrentRound)
```

Comment:
- timeout / vote we have different rules. perhaps this err can print more accurate string.
- chaning this also changes `TestThrowErrorIfTimeoutMsgRoundNotEqualToCurrentRound` and `TestTimeoutHandlerRoundNotEqual`